server/scripts/import-tests/index.old.js

```diff
@@ -0,0 +1,446 @@
+/* eslint no-console: 0 */
```

Comment:
If this is just a copy of the old version I think we can delete this and go back to the Git history if we ever need it, what do you think?